lib/healthcheck/healthcheck-control.c

```diff
@@ -27,11 +27,39 @@
 #include "control/control.h"
 #include "control/control-connection.h"
 #include "control/control-commands.h"
+#include "stats/stats-prometheus.h"
+
+static void
+_send_healthcheck_reply(HealthCheckResult result, gpointer c)
+{
+  ControlConnection *cc = (ControlConnection *) c;
+
+  GString *reply = g_string_new("OK ");
+
+  g_string_append_printf(reply, PROMETHEUS_METRIC_PREFIX
+                         "io_worker_latency_nanoseconds %" G_GUINT64_FORMAT"\n",
```

Comment:
prometheus [recommends](https://prometheus.io/docs/practices/naming/) to use base units, e.g. **seconds**:
> ...should use base units (e.g. seconds, bytes, meters - not milliseconds, megabytes, kilometers). See below for a list of base units.

Reply:
Yeah. Unfortunately, representing this as a `double` is not precise enough (it showed 0 in most cases), so I decided to use nanoseconds anyway.

Reply:
I don't think it would make sense as Prometheus will store it the same way as we would, losing precision. But let me know if you disagree.

Reply:
We've talked about this IRL. I'll double-check the precision issue and try implementing "unit" support for stats.

Reply:
I've added unit of measure support.